nerfstudio/data/dataparsers/arkitscenes_dataparser.py

```diff
@@ -32,10 +32,10 @@
 
 
 # Taken from https://github.com/apple/ARKitScenes/blob/main/threedod/benchmark_scripts/utils/tenFpsDataLoader.py
-def traj_strin_to_matrix(traj_str: str):
-    """convert traj_str into translation and rotation matrices
+def traj_strin_to_matrix(traj_string: str):
```

Comment:
Ahh, I was referring to the typo in the function name.

Reply:
Ok, let me quickly fix!

Reply:
Ok, fixed in e279656454a7d5be5c340ca9e0eac111309b9fed.